lte/gateway/fabfile.py

```diff
@@ -391,6 +391,7 @@ def bazel_integ_test_post_build(
     else:
         ansible_setup(test_host, "test", "magma_test.yml")
 
+    run('while [[ 0 ]]; do sleep 2 && top -b -n 1 | grep Cpu && free -h; done &')
```

Comment:
I'm a bit confused about this. This is supposed to keep running in the background to print cpu and memory statistics?

A comment would be helpful.

Reply:
this was pushed by accident - already removed